pkg/reboot/event.go

```diff
@@ -0,0 +1,62 @@
+package reboot
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/leptonai/gpud/components"
+	"github.com/leptonai/gpud/pkg/common"
+	"github.com/leptonai/gpud/pkg/eventstore"
+	"github.com/leptonai/gpud/pkg/log"
+)
+
+func RecordEvent(ctx context.Context, eventStore eventstore.Store, bucketName string, lastRebootTime func(context.Context) (time.Time, error)) error {
```

Comment:
Let's try not to pass in function references, especially if it's just for testing. 
We should make the function `RecordEvent` private and should add functions like `UpdateOSRebootTime` or `SyncOSRebootTime`

```suggestion
func RecordEvent(ctx context.Context, eventStore eventstore.Store, bucketName string) error {
```

Reply:
@eahydra Done